site/docs/v1/tech/apis/_form-request-body.adoc

```diff
@@ -5,11 +5,44 @@
 [field]#form.data# [type]#[Object]# [optional]#Optional#::
 An object that can hold any information about the Form that should be persisted.
 
+[field]#form.type# [type]#[String]# [optional]#Optional# [default]#defaults to `registration`# [since]#Available since 1.20.0#::
+The type of form being created, a form type cannot be changed after the form has been created. This type will be used to identify how this form can be utilized by FusionAuth.
++
+ * `registration` This form will be used for self service registration.
+ * `adminRegistration` This form be used to customize the add and edit User Registration form in the FusionAuth UI.
+ * `adminUser` This form can be used to customize the add and edit User form in the FusionAuth UI.
```

Comment:
Add since annotations, should also probably add a dash between the type and description. See usages of `[field]#authenticationType#` for example.